api/base/authentication/drf.py

```diff
@@ -51,7 +51,10 @@ def authenticate_credentials(self, userid, password):
         return (user, None)
 
     def authenticate_header(self, request):
-        return ""
+        """
+        Returns custom value other than "Basic" to prevent BasicAuth dialog prompt when returning 401
+        """
+        return 'Documentation realm="{}"'.format(self.www_authenticate_realm)
```

Comment:
I'm not quite sure on this approach. We're technically sending incorrect headers for Basic Auth, but I think the cost-benefit ratio favors doing this to make the browse-able API more usable. I think this is fine for now.